.github/workflows/frontend-update-static-flags.yml

```diff
@@ -0,0 +1,53 @@
+name: Frontend Update Static Flags
+
+on: [push, pull_request]
+#on:
+#  schedule:
+#    - cron: 0 8 * * *
+
+defaults:
+  run:
+    working-directory: frontend
+
+jobs:
+  check-static-flags-updated:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cloning repo
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - name: Cache dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/.npm
+          key: npm-${{ hashFiles('package-lock.json') }}
+          restore-keys: npm-
+
+      - name: Pull Updated Static Flags
+        id: get_changes
+        run: |
+          npm ci
+          npm run static:generate
+          echo "changed=$(git status --porcelain | wc -l)" >> $GITHUB_OUTPUT
+
+      - name: Create Pull Request for new flags if required
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_PRIVATE_ACCESS_TOKEN }}
+        if: steps.get_changes.outputs.changed != 0
+        run: |
+          git config user.email "ben.rometsch@flagsmith.com"
+          git config user.name "Ben Rometsch"
+          git stash
```

Comment:
Do we need the `stash` and `stash pop` ? As far as I can tell, `git checkout -B <branch name>` doesn't overwrite local changes.